front_end/entrypoints/inspector_main/BUILD.gn

```diff
@@ -23,7 +23,6 @@ devtools_module("inspector_main") {
     "../../panels/mobile_throttling:bundle",
     "../../ui/legacy:bundle",
     "../../ui/legacy/components/utils:bundle",
-    "../../ui/components/buttons:bundle",
```

Comment:
We should not remove that because it produces an error when building the Inspector with ninja
![image](https://user-images.githubusercontent.com/29014573/226648909-bf302b4b-15db-4fee-a1a7-8ccd095ccc97.png)